src/www/ui_components/app-root.js

```diff
@@ -694,7 +694,7 @@ export class AppRoot extends mixinBehaviors
   }
 
   _computeShouldShowQuitButton(platform) {
-    return platform === 'Mac OS X' || platform === 'Electron';
+    return platform === 'Electron';
```

Comment:
Why are you removing the quit button? How would someone quit the app?

Reply:
The button was not showing up. I fixed it by updating the macos platform string to "macOS" (formerly "Mac OS X"), which comes from cordova-plugin-device. 

> How would someone quit the app?


Cmd + Q